corehq/apps/reports/standard/deployments.py

```diff
@@ -234,6 +248,44 @@ def process_rows(self, users, fmt_for_export=False):
             ])
         return rows
 
+    def process_users(self, users, fmt_for_export=False):
```

Comment:
is fmt_for_export passed in from any caller?

Reply:
currently no, but when I rewrite the export code to use this backend it will need it, so i thought it was worth including up front. Plus it matches the interface of the `process_rows` function which is the analogous function for the ES backend. happy to drop it for now though